src/shadowbox/docker/run_action.sh

```diff
@@ -17,12 +17,16 @@
 do_action shadowbox/docker/build
 
 readonly OUTLINE_DIR=/tmp/outline
-mkdir -p $OUTLINE_DIR && touch "$OUTLINE_DIR/config.json"
+readonly STATE_DIR=$OUTLINE_DIR/persisted-state
+readonly STATE_CONFIG=$STATE_DIR/shadowbox_server_config.json
+mkdir -p $STATE_DIR && touch "$OUTLINE_DIR/config.json"
+[[ -e $STATE_CONFIG ]] || echo "{\"hostname\":\"127.0.0.1\", \"apiPort\":2357}" > $STATE_CONFIG
 source $ROOT_DIR/src/shadowbox/scripts/make_test_certificate.sh "${OUTLINE_DIR}"
 
 # TODO: mount a folder rather than individual files.
 declare -a docker_bindings=(
   -v "$OUTLINE_DIR/config.json":/root/shadowbox/shadowbox_config.json
```

Comment:
Let's use shadowbox_config.json instead of config.json. It's less confusing.

Reply:
In fact, this line is not needed if you are mounting the state dir. Both config files go there.

Reply:
See what we do in production: https://github.com/Jigsaw-Code/outline-server/blob/47446d4c8bf5a3c6965c26a66b33a9c80914f57c/src/server_manager/install_scripts/install_server.sh#L262-L272

We actually mount the state dir. The config files are in it. Let's reproduce that, so it's closer to production and easier to understand. Using a different structure and file names for dev is confusing.